apps/fortuna/src/keeper.rs

```diff
@@ -499,6 +544,34 @@ pub async fn process_single_block_batch(
             Ok(events) => {
                 tracing::info!(num_of_events = &events.len(), "Processing",);
                 for event in &events {
```

Comment:
can you add a TODO here that this is handling all events in a single block sequentially? We might need to parallelise this later on.